scripts/relabel_nnue_results.py

```
@@ -0,0 +1,95 @@
+    # Pass one is deliberately read-only. This makes the operation idempotent
+    # in the safe sense: an already-correct or malformed corpus is rejected
+    # before any bytes are changed.
+    with path.open("rb") as stream:
+        while True:
+            line = stream.readline()
+            if not line:
+                break
+
+            parts = line.rstrip(b"\r\n").rsplit(b" | ", 2)
+            if len(parts) != 3:
+                raise ValueError(f"{path}:{records + 1}: expected FEN | score | result")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)